client/me/help/help-contact/index.jsx

```diff
@@ -582,16 +598,27 @@ const HelpContact = React.createClass( {
 		return SUPPORT_HAPPYCHAT !== variationSlug && SUPPORT_LIVECHAT !== variationSlug && null != ticketSupportRequestError;
 	},
 
-	shouldShowPreloadForm: function() {
-		const { olark, sitesInitialized } = this.state;
+	/**
+	 * Before determining which variation to assign, certain async data needs to be in place.
+	 * This function helps assess whether we're ready to say which variation the user should see.
+	 *
+	 * @returns {Boolean} Whether all the data is present to determine the variation to show
+	 */
+	hasDataToDetermineVariation: function() {
```

Comment:
@jordwest Actually I'm gonna punt on this for now. I started making the change and realized that it would be a significant-enough alteration (more than just extracting existing logic) that I'd want to do a lot more testing and peer review. Let's save this for cleanup when we 💣 Olark.